platform/kobo/koreader.sh

```diff
@@ -80,6 +80,9 @@ if [ ! -n "${PLATFORM}" ] ; then
 fi
 # end of value check of PLATFORM
 
+# Remount SD to RW, ignore errors since we may not have sd card
```

Comment:
What about my second suggestion regarding checking whether it is in fact already mounted using either mount itself or e.g. cat /proc/mounts?